### ISSUE TITLE
Breadcrumb colour contrast not sufficient

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#180]
Contrast between foreground text and background must be increases to meet WCAG AA standards

<img width="1100" alt="Screen Shot 2020-04-14 at 11 53 17 AM" src="https://user-images.githubusercontent.com/31868510/79246126-0cc16480-7e47-11ea-9555-225ccc6cdddf.png">

Element has insufficient color contrast of 4.47 (foreground color: #777777, background color: #ffffff, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1
